index.html

```diff
@@ -0,0 +1,96 @@
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="https://momentui.netlify.app/css/components.css" />
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+		integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
+		crossorigin="anonymous" referrerpolicy="no-referrer" />
+	<link rel="stylesheet" href="./css/main.css">
+	<title>Home | Quiz App</title>
+</head>
+
+<body>
+	<nav class="navbar nav-ecommerce">
+		<div class="navbar-section">
+			<button class="nav-menu-btn">
+				<span class="material-icons">menu</span>
+			</button>
+			<a class="navbar-brand" href="/index.html">Kool Quiz</a>
+		</div>
+		<div class="navbar-section">
+			<ul class="navbar-nav">
+				<li class="nav-item">
+					<a href="#" class="nav-link">
+						<i class="fas fa-moon" aria-hidden="true"></i>
+					</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link">
+						<i class="material-icons-outlined">
+							person
+						</i>
+					</a>
+				</li>
+			</ul>
+			<button class="btn btn-primary btn-sm login-btn">Login</button>
+		</div>
+	</nav>
+	<main class="main-wrapper">
+		<section class="section-wrapper container">
+			<h2 class="text-left category-title font-weight-500">Featured Categories</h2>
+			<div class="featured-categories-container">
+				<div class="grid grid-col-1">
+					<div class="featured-categories-card">
+						<div class="card quiz-card card-shadow">
+							<div class="card-header">
+								<img class="img-responsive"
+									src="https://i0.wp.com/www.opindia.com/wp-content/uploads/2022/01/Untitled-design-13.jpg?fit=700%2C400&ssl=1"
+									alt="..">
+							</div>
+							<div class="card-body">
+								<div class="text-badge">Entertainment</div>
+								<h5 class="card-title">Shark Tank India</h5>
+								<p class="card-description">Play this fun quiz to test your expertise! </p>
+								<button class="btn btn-play">Play</button>
+							</div>
+						</div>
+					</div>
+					<div class="featured-categories-card">
+						<div class="card quiz-card card-shadow">
+							<div class="card-header">
+								<img class="img-responsive"
+									src="https://images.news18.com/ibnlive/uploads/2021/08/national-flag-16289132954x3.jpg"
+									alt="..">
+							</div>
+							<div class="card-body">
+								<div class="text-badge">Country</div>
+								<h5 class="card-title">Chak De India</h5>
+								<p class="card-description">Test your general knowledge about India. </p>
+								<button class="btn btn-play">Play</button>
+							</div>
+						</div>
+					</div>
+					<div class="featured-categories-card">
+						<div class="card quiz-card card-shadow">
+							<div class="card-header">
+								<img class="img-responsive" src="https://reactjs.org/logo-og.png" alt="..">
+							</div>
```

Comment:
can we add a better description in alt instead of .. ?

Reply:
Done. Thank you